inference/deepspeed_predictor.py

```diff
@@ -238,13 +238,16 @@ def streaming_generate(self, prompt, streamer, **config):
         for worker in self.prediction_workers[1:]:
             worker.streaming_generate.remote(inputs_ref, self._create_dummy_streamer(), **config)
 
-    def generate(self, prompt, **config):
+    def generate(self, prompt, return_shape=False, **config):
```

Comment:
The generate interface is an override of Predictor interface. We should have consistency maintained across interfaces.
I don't' think it's a good idea to change the interface just for the return value.

Reply:
If the shape value is returned directly without passing parameter, we need to modify the previously called code.